config/runtime.exs

```diff
@@ -36,7 +36,10 @@ if config_env() != :test do
     message_log_s3_folder: System.get_env("MESSAGE_LOG_S3_FOLDER"),
     message_log_report_s3_folder: System.get_env("MESSAGE_LOG_REPORT_S3_FOLDER"),
     monitoring_api_key: System.get_env("MONITORING_API_KEY"),
-    active_headend_path: System.get_env("ACTIVE_HEADEND_S3_PATH")
+    active_headend_path: System.get_env("ACTIVE_HEADEND_S3_PATH"),
+    screenplay_url: System.get_env("SCREENPLAY_URL"),
+    screenplay_api_key: System.get_env("SCREENPLAY_API_KEY"),
+    active_pa_messages_path: System.get_env("ACTIVE_PA_MESSAGES_PATH")
```

Comment:
Hmm yeah that's a good point. I think my rationale was just that the path could change since the screenplay api is relatively new and still being developed, but I guess it would be just as much effort to make the devops change rather than just make a code change in RTS or probably even less. I'll make a change to remove this one